packages/transaction-controller/src/constants.ts

```diff
@@ -20,7 +20,15 @@ export const CHAIN_IDS = {
   MOONBEAM_TESTNET: '0x507',
   MOONRIVER: '0x505',
   GNOSIS: '0x64',
+  ARBITRUM: '0xa4b1',
+  ZKSYNC_ERA: '0x144',
 } as const;
+const SWAPS_TESTNET_CHAIN_ID = '0x539';
```

Comment:
Minor but new line between constants?

Reply:
Fixed in https://github.com/MetaMask/core/pull/1877/commits/7b7df50a8304dd7c976963574eb69f95af6feb11